docs/sql-data-sources-jdbc.md

```diff
@@ -229,9 +229,9 @@ logging into the data sources.
     <td><code>truncate</code></td>
     <td><code>false</code></td>
     <td>
-      This is a JDBC writer related option. When <code>SaveMode.Overwrite</code> is enabled, this option causes Spark to truncate an existing table instead of dropping and recreating it. This can be more efficient, and prevents the table metadata (e.g., indices) from being removed. However, it will not work in some cases, such as when the new data has a different schema. In case of failures, users should turn off <code>truncate</code> option to use <code>DROP TABLE</code> again. Also, due to the different behavior of <code>TRUNCATE TABLE</code> among DBMS, it's not always safe to use this. MySQLDialect, DB2Dialect, MsSqlServerDialect, DerbyDialect, and OracleDialect supports this while PostgresDialect and default JDBCDirect doesn't. For unknown and unsupported JDBCDirect, the user option <code>truncate</code> is ignored.
```

Comment:
While touching this line, shall we change `among DBMS` -> `among DBMSes`?

Reply:
Yes, this is Ok.